boa_engine/src/vm/mod.rs

```diff
@@ -182,6 +182,7 @@ impl Context {
 
         while self.vm.frame().pc < self.vm.frame().code.code.len() {
             #[cfg(feature = "fuzz")]
+            #[allow(clippy::redundant_else)]
             if self.instructions_remaining == 0 {
                 return Err(JsError::from_native(JsNativeError::no_instructions_remain()));
             } else {
```

Comment:
Can we reformat this code a bit to not need to allow the lint? I see two options:
 - We can invert and use a `!= 0` in the first condition, and `return` in the `else`
 - We can use a explicit scope, and not have the `else`

I'm personally more towards the second one, since it clearly marks which code will only be compiled for the "fuzz" feature.

Reply:
Done